### PR TITLE
Reverse the string

### PR DESCRIPTION
**Pull request by AmrutaDhage20**

### CODE REVIEW

File: src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            }
+        }
+        if (position != -1) {
+            reverse.insert(position, '@');
+        }
+        System.out.println(reverse.toString());
+        return reverse.toString();
+    }
+    public static void main(String[] args) {
+        String myString = stringReverse("Am@ruta");
```
Comment (amitsahu8) [inline src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java]: input does not match with readme

**Review by amitsahu8 — CHANGES_REQUESTED** [review]

**Review by amitsahu8 — APPROVED** [review]

File: src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    public static String stringReverse(String myString) {
+        StringBuilder str = new StringBuilder(myString);
+        StringBuilder reverse = new StringBuilder();
+        int position = -1;
+        for (int i = 0; i < str.length(); i++) {
+            if (str.charAt(i) != '@') {
+                reverse.insert(0, str.charAt(i));
+            } else {
+                position = i;
+                System.out.println(i);
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java]: remove this print statement

File: src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            }
+        }
+        if (position != -1) {
+            reverse.insert(position, '@');
+        }
+        System.out.println(reverse.toString());
+        return reverse.toString();
+    }
+    public static void main(String[] args) {
+        String myString = stringReverse("Ro@hitshetty");
```
Comment (prashantkawa) [inline src/com/encora/javaAssignments/MediumComplexityProblems/reverseString/ReverseString.java]: No use of this variable

**Review by prashantkawa — CHANGES_REQUESTED** [review]

**Review by prashantkawa — APPROVED** [review]
Approving PR

**Pull request merged**